packages/web/src/scss/components/Card/_CardMedia.scss

```diff
@@ -0,0 +1,82 @@
+// 1. Expand any images or videos to fill the available space, not just direct descendants. This is for the scenario
+//    when the media is wrapped, for example in a link.
+// 2. Make sure rounded corners are applied to video player too.
+// 3. Do not let the media push the body content out of the horizontal card.
+// 4. We need the specific combination of classes, expanded CardMedia is designed to work only with boxed cards.
+
+@use '@tokens' as tokens;
+@use '../../tools/dictionaries';
+@use 'theme';
+
+.CardMedia {
+    --#{tokens.$css-variable-prefix}card-media-border-radius-top-left: #{theme.$border-radius};
+    --#{tokens.$css-variable-prefix}card-media-border-radius-top-right: #{theme.$border-radius};
+    --#{tokens.$css-variable-prefix}card-media-border-radius-bottom-left: #{theme.$border-radius};
+    --#{tokens.$css-variable-prefix}card-media-border-radius-bottom-right: #{theme.$border-radius};
+    --#{tokens.$css-variable-prefix}card-media-canvas-width: auto;
+    --#{tokens.$css-variable-prefix}card-media-canvas-height: auto;
+
+    grid-area: media;
+}
+
+.CardMedia__canvas {
+    width: var(--#{tokens.$css-variable-prefix}card-media-canvas-width);
+    max-width: 100%; // 3.
+    height: var(--#{tokens.$css-variable-prefix}card-media-canvas-height);
+    overflow: hidden;
+    border-radius: var(--#{tokens.$css-variable-prefix}card-media-border-radius-top-left)
```

Comment:
I'm considering moving over these two properties to `.CardMedia` so `CardMedia__canvas` can be easily extracted as `Media` later on. It's easier to do now so it won't be a BC in the `web` package.

@crishpeen What do you think?